infra/templates/az-isolated-service-single-region/tests/unit/unit_test.go

```diff
@@ -24,6 +24,20 @@ var tfOptions = &terraform.Options{
 	TerraformDir: "../../",
 	Upgrade:      true,
 	Vars: map[string]interface{}{
+		"resource_ip_whitelist": []string{
```

Comment:
Had a brief discussion with @KeithJRome and @ThePhen and we decided to drop the ip whitelist to 2 ip addresses as these IP's are specific to Azure zones.

Reply:
I'm changing this list to two arbitrary IP addresses. For the purposes of this unit test, the IP values aren't important, it is just important that these values appear in the `tf plan` output.